lib/matplotlib/tests/test_figure.py

```diff
@@ -861,3 +861,48 @@ def test_subfigure_ss():
     ax.set_title('Axes')
 
     fig.suptitle('Figure suptitle', fontsize='xx-large')
+
+
+@image_comparison(['test_subfigure_double.png'], style='mpl20',
+                  savefig_kwarg={'facecolor': 'teal'},
+                  remove_text=False)
+def test_subfigure_double():
+    # test assigning the subfigure via subplotspec
+    np.random.seed(19680808)
+    def example_plot(ax, fontsize=12, hide_labels=False):
```

Comment:
`hide_labels` is never `True`, and this is only called once; maybe it should be embedded, and then `nn` could be used in the loop?